docs/notebooks/unfinished/example_measure_elzerman_readout.py

```diff
@@ -0,0 +1,54 @@
+from qtt.measurements.acquisition import UHFLIScopeReader, UHFLIStimulus
```

Comment:
This example script cannot be executed without the UHFLI hardware. If this is converted to a jupyter notebook, then the output that is generated by the script can also be included. E.g. the output of `print(records)`

Would it be an option to do that?